sigstore/_internal/set.py

```diff
@@ -2,8 +2,52 @@
 Utilities for verifying Signed Entry Timestamps.
 """
 
+import base64
+from importlib import resources
+from typing import cast
+
+import cryptography.hazmat.primitives.asymmetric.ec as ec
+import jcs  # type: ignore
+from cryptography.exceptions import InvalidSignature
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.serialization import load_pem_public_key
+
 from sigstore._internal.rekor import RekorEntry
 
+REKOR_ROOT_PUBKEY = resources.read_binary("sigstore._store", "rekor.pub")
```

Comment:
I will allow the user to provide these (any default to using the ones in the package) in a separate PR since we're doing this in a few places. Let's do it like this for now.